packages/create-mc-app/src/cli.ts

```diff
@@ -0,0 +1,109 @@
+import { cac } from 'cac';
+import { Listr, ListrTask } from 'listr2';
```

Comment:
It's more a question than a suggestion - I was wondering if you'd see any downside of importing `ListrTask` interface as type here? (It's also imported in a few other places). I'm asking since it seems to me it was done like this for a reason.

Reply:
We should use `type ListrTask` yes, thanks for pointing it out.